src/main/java/gift/entity/Order.java

```diff
@@ -0,0 +1,44 @@
+package gift.entity;
+
+import jakarta.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private int quantity;
+
+    private String message;
+
+    private LocalDateTime orderDateTime;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id", nullable = false)
+    private Member member;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "option_id", nullable = false)
+    private ProductOption option;
+
+    protected Order() {}
+
+    public Order(Member member, ProductOption option, int quantity, String message) {
+        this.member = member;
+        this.option = option;
+        this.quantity = quantity;
+        this.message = message;
+        this.orderDateTime = LocalDateTime.now();
+    }
+
+    public Long getId() { return id; }
+    public int getQuantity() { return quantity; }
+    public String getMessage() { return message; }
+    public LocalDateTime getOrderDateTime() { return orderDateTime; }
+    public Member getMember() { return member; }
+    public ProductOption getOption() { return option; }
```

Comment:
한줄로 표현할 수 있더라도 가능한 `{`, `}`은 줄바꿈을 통해서 가독성을 확보해주시면 좋을 것 같아요. 
(IDE의 자동 정렬을 진행하셔보셔도 좋을 것 같네요. 😃) 

[구글 자바 코드 컨벤션](https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style)에서도 `{` 시작 과 `}`끝에 줄바꿈을 넣으라는 가이드를하고있어요. 

```suggestion
     public Long getId() {
        return id;
    }

    public int getQuantity() {
        return quantity;
    }

    public String getMessage() {
        return message;
    }

    public LocalDateTime getOrderDateTime() {
        return orderDateTime;
    }

    public Member getMember() {
        return member;
    }

    public ProductOption getOption() {
        return option;
    }
```